physionet-django/old/templates/footer.html

```diff
@@ -0,0 +1,18 @@
+<!-- This footer template is from the bootstrap 'blog' example -->
```

Comment:
@fpdrozd What is the purpose of the "old" folder in these paths (e.g. `physionet-django/old/templates/footer.html`)? If this content is no longer needed, can't it just be deleted?